### PR TITLE
Add a note regarding default db permissions

### PR DESCRIPTION
**Pull request by Tener**

### CODE REVIEW

**Comment by Tener** [comment]
Link to changed part of the doc: https://docs-mova8rs88-goteleport.vercel.app/docs/database-access/auto-user-provisioning/mysql/#step-33-connect-to-the-database

File: docs/pages/database-access/auto-user-provisioning/mysql.mdx [hunk 1]
```diff
@@ -97,6 +97,18 @@ database queries in the Teleport Audit Logs, when the Teleport username is over
 
 ## Step 3/3. Connect to the database
 
+<Admonition type="note" title="Database permissions">
+By default, the newly created won't have permissions to access any particular database. These permissions must be
+granted either through the common role `teleport-auto-user` or through database-specific roles, such as `reader`.
```
Comment (greedy52) [inline docs/pages/database-access/auto-user-provisioning/mysql.mdx:102]: I would not recommend granting permissions to `teleport-auto-user` as the role is created by `teleport-admin` and the role is only meant for tracking teleport-managed users.

If my understanding is correct, if any of the roles assigned to the auto-provisioned user has some meaningful grant, `tsh db connect` should succeed on the db-name that the role grants? I would either convert this block to a Troubleshooting or a Prerequisites. What do you think?

Comment (Tener) [reply docs/pages/database-access/auto-user-provisioning/mysql.mdx:102]: Sure, that works. Although if `teleport-auto-user` shouldn't be touched, then in practice does it mean the user should have at least one other role?

Comment (greedy52) [reply docs/pages/database-access/auto-user-provisioning/mysql.mdx]: Yes. We can expect at least one `db_roles` to be assigned through the Teleport role (or roles). Though seems we are not enforcing this (like not checking against empty `db_roles`)

File: docs/pages/database-access/auto-user-provisioning/mysql.mdx [hunk 2]
```diff
@@ -99,6 +99,18 @@ database queries in the Teleport Audit Logs, when the Teleport username is over
 
 (!docs/pages/includes/database-access/auto-user-provisioning/connect.mdx gui="MySQL Workbench"!)
 
+## Troubleshooting
+
+By default, the newly created won't have permissions to access any particular database. These permissions should be
```
Comment (ptgott) [inline docs/pages/database-access/auto-user-provisioning/mysql.mdx:104]:
```suggestion
By default, the newly created users won't have permissions to access any particular database. These permissions should be
```

File: docs/pages/database-access/auto-user-provisioning/mysql.mdx [hunk 3]
```diff
@@ -99,6 +99,18 @@ database queries in the Teleport Audit Logs, when the Teleport username is over
...
+granted through database-specific roles, such as `reader`.
+
+Otherwise you may see errors like follows:
```
Comment (ptgott) [inline docs/pages/database-access/auto-user-provisioning/mysql.mdx:107]:
```suggestion
Otherwise you may see errors like the following:
```

**Review by ptgott — APPROVED** [review]

**Comment by ptgott** [comment]
Are we backporting this one?

File: docs/pages/database-access/auto-user-provisioning/mysql.mdx (hunk 1 above)
Comment (Tener) [reply docs/pages/database-access/auto-user-provisioning/mysql.mdx]: There might be some cases where this makes sense, so perhaps we don't need to change the requirements here. I think the docs are fine now, going to merge in a bit.

**Review by greedy52 — APPROVED** [review]

**Pull request merged**